.github/workflows/pr-validation.yml

```diff
@@ -9,23 +9,26 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        dotnetVersion: [3.1.x, 5.x, 6.x, 7.x, 8.x]
+        dotnetVersion: [6.x, 7.x, 8.x, 9.x]
```

Comment:
Why are you adding .NET 7, it is out of support?

Reply:
.NET 7 is still supported so we'll keep it there for now

Reply:
I dont think so: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

Reply:
Oh wow looks like it just ended yesterday, although I don't think there's any harm in testing for NET 7 too for now.

Reply:
why not removing net7 if it's out of support?